src/middlewared/middlewared/plugins/smb_/registry_share.py

```diff
@@ -66,7 +66,7 @@ async def netconf(self, **kwargs):
 
     @private
     async def reg_listshares(self):
```

Comment:
When creating a fresh SMB share using the `cluster_volname` schema attribute, I was presented with this traceback. The share was created successfully, just raised that exception.

```  File "/usr/lib/python3/dist-packages/middlewared/plugins/smb.py", line 935, in do_create
    ret = await self.query([('name', '=', data['name'])],
  File "/usr/lib/python3/dist-packages/middlewared/schema.py", line 998, in nf
    return await f(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/middlewared/plugins/smb.py", line 1130, in query
    result = await self.middleware.call(
  File "/usr/lib/python3/dist-packages/middlewared/main.py", line 1263, in call
    return await self._call(
  File "/usr/lib/python3/dist-packages/middlewared/main.py", line 1220, in _call
    return await methodobj(*prepared_call.args)
  File "/usr/lib/python3/dist-packages/middlewared/schema.py", line 998, in nf
    return await f(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/middlewared/plugins/smb_/registry_share.py", line 287, in registry_query
    return filter_list(rv, filters, options)
  File "/usr/lib/python3/dist-packages/middlewared/utils/__init__.py", line 198, in filter_list
    raise MatchNotFound()
middlewared.service_exception.MatchNotFound